dm/master/server.go

```diff
@@ -1284,7 +1275,7 @@ func (s *Server) fetchWorkerDDLInfo(ctx context.Context) {
 				default:
 					resp, err := cli.SendRequest(ctx, request, s.cfg.RPCTimeout)
 					if err != nil {
-						log.L().Error("create FetchDDLInfo stream", zap.String("worker", worker), log.ShortError(err))
+						log.L().Error("create FetchDDLInfo stream", zap.String("worker", source), log.ShortError(err))
```

Comment:
```suggestion
						log.L().Error("create FetchDDLInfo stream", zap.String("source", source), log.ShortError(err))
```

ditto for the following lines.

Reply:
ok.